utils/sl/components/__init__.py

```diff
@@ -312,7 +312,7 @@ def define_components(reqs):
                 libs=['rte_bus_pci'], patch_rpath=['lib'])
 
     url = 'https://github.com/protobuf-c/protobuf-c/releases/download/' \
-        'v1.3.3/protobuf-c-1.3.3.tar.gz'
+        'v1.3.0/protobuf-c-1.3.0.tar.gz'
```

Comment:
I'm a little confused by this -- why the downgrade? I thought we discussed making 1.3.3 the minimum version?

Reply:
apologies I forgot to update the comments regarding this, the upgrade is causing problems in build so I've decided to move it to a stand-alone PR to separate concerns

Reply:
this change just reverts to existing behaviour and build